src/main/java/com/cinema/model/Ticket.java

```diff
@@ -0,0 +1,74 @@
+package com.cinema.model;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "tickets")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "ticket_id")
+    private Long id;
+    @ManyToOne (fetch = FetchType.LAZY)
```

Comment:
why are you using FetchType.LAZY here and below? and also in MovieSession class?

it is not a big deal if we will fetch only one more related object along with the main one

that's why FetchType.EAGER is default for `@ManyToOne` and `@OneToOne` relations

Reply:
I was training "join fetch" - if not lazy, no point to do join fetch